src/sonic-yang-models/yang-models/sonic-crm.yang

```diff
@@ -293,6 +293,66 @@ module sonic-crm {
                     type threshold;
                 }
 
+                leaf dnat_entry_threshold_type {
```

Comment:
Changes looks good to me, but we need to add tests.

-------------
To add Test for sonic-yang-module: 

1.) Add example Config at:
https://github.com/Azure/sonic-buildimage/blob/e52581e919c95e861cf630c9c7e1a840fc5e8ebd/src/sonic-yang-models/tests/yang_model_tests/yangTest.json#L2


Example config should be in sonic-yang format.

2.) Add matching detail and expected error string at:
https://github.com/Azure/sonic-buildimage/blob/e52581e919c95e861cf630c9c7e1a840fc5e8ebd/src/sonic-yang-models/tests/yang_model_tests/test_yang_model.py#L52


If You add new leaf in YANG Models:
Then add corresponding config in same format as in configDB.json at:
https://github.com/Azure/sonic-buildimage/blob/4cf9316ec349f80deceb0d7665b0aaf85bfe2599/src/sonic-yang-models/tests/yang_model_tests/yangTest.json#L1038


And build both:
sonic_yang_mgmt-1.0-py3-none-any.whl
sonic_yang_models-1.0-py3-none-any.whl

Build tests will make sure, Translation works for new leafs in YANG models.

-------------